docs/exts/docs_build/lint_checks.py

```diff
@@ -230,13 +230,13 @@ def find_modules(deprecated_only: bool = False) -> Set[str]:
 
 
 def check_exampleinclude_for_example_dags() -> List[DocBuildError]:
-    """Checks all exampleincludes for  example dags."""
-    all_docs_files = glob(f"${DOCS_DIR}/**/*.rst", recursive=True)
+    """Checks all exampleincludes for example dags."""
```

Comment:
"exampleincludes"?, missed space?

Reply:
I guess not, it is how it looks in the docs:
`.. exampleinclude::`

Reply:
ok, thx